schema

```diff
@@ -0,0 +1,243 @@
+{
+  "id": "http://jsonapi.org/schema#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "JSON API Schema",
+  "description": "This is a schema for responses in the JSON API format. For more, see http://jsonapi.org",
+  "oneOf": [
+    { "$ref": "#/definitions/success" },
+    { "$ref": "#/definitions/failure" }
+  ],
+  "definitions": {
+    "success": {
+      "type": "object",
+      "required": [ "data" ],
+      "properties": {
+        "meta": { "$ref": "#/definitions/meta" },
+        "data": { "$ref": "#/definitions/data" },
+        "included": { "$ref": "#/definitions/included" },
+        "links": { "$ref": "#/definitions/topLevelLinks" }
+      },
+      "additionalProperties": false
+    },
+    "failure": {
+      "type": "object",
+      "required": [ "errors" ],
+      "properties": {
+        "meta": { "$ref": "#/definitions/meta" },
+        "errors": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/error" },
+          "uniqueItems": true
+        }
+      },
+      "additionalProperties": false
+    },
+
+    "meta": {
+      "description": "Non-standard meta-information that can not be represented as an attribute or relationship.",
+      "type": "object",
+      "additionalProperties": true
+    },
+    "data": {
+      "description": "Primary data **MUST** appear under a top-level key named \"data\". Primary data **MUST** be either a single resource object, an array of resource objects, or a value representing a resource relationship.",
+      "oneOf": [
+        { "$ref": "#/definitions/resource" },
+        {
+          "description": "An array of resource objects.",
+          "type": "array",
+          "items": { "$ref": "#/definitions/resource" },
+          "uniqueItems": true
+        }
+      ]
+    },
+    "resource": {
+      "description": "\"Resource objects\" appear in a JSON API document to represent primary data and linked resources.",
+      "type": "object",
+      "required": [ "type", "id" ],
+      "properties": {
+        "type": { "type": "string" },
+        "id": { "type": "string" },
+        "links": { "$ref": "#/definitions/links" },
+        "meta": { "$ref": "#/definitions/meta" }
+      },
+      "additionalProperties": true
```

Comment:
There should be attributes and relationships definitions here, and then additionalProperties should be false